runtime/src/Provider.tsx

```diff
@@ -8,7 +8,7 @@ import React from 'react'
 
 type ProviderInput = {
     config: Config
-    userInfo: CurrentUser
+    userInfo?: CurrentUser
```

Comment:
small nit: I think it's more accurate to have the type as `CurrentUser | undefined`, right? - as in `userInfo` will always be there, but might be undefined?